litellm/integrations/opentelemetry.py

```diff
@@ -30,6 +30,9 @@
 LITELLM_RESOURCE = {
     "service.name": os.getenv("OTEL_SERVICE_NAME", "litellm"),
 }
+LITELLM_ENVIRONMENT = {
+    "deployment.environment": os.getenv("OTEL_ENVIRONMENT_NAME", "production"),
+}
```

Comment:
Can you change to 

```
LITELLM_RESOURCE = {
    "service.name": os.getenv("OTEL_SERVICE_NAME", "litellm"),
    "deployment.environment": os.getenv("OTEL_ENVIRONMENT_NAME", "production"),
}
```

and then drop `LITELLM_DEPLOYMENET`

Reply:
done